zebra/table_manager.h

```diff
@@ -57,15 +57,19 @@ struct table_manager_chunk {
  */
 struct table_manager {
 	struct list *lc_list;
+	uint32_t start;
+	uint32_t end;
 };
 
-void table_manager_enable(ns_id_t ns_id);
+void table_manager_enable(struct zebra_vrf *zvrf);
 struct table_manager_chunk *assign_table_chunk(uint8_t proto, uint16_t instance,
-					       uint32_t size);
+					       uint32_t size, vrf_id_t vrf_id);
```

Comment:
why does `assign` take a vrf id, but `release` takes a vrf object pointer - that seems ... asymmetrical?